spec/order_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'order.rb'
+
+# I would like to be able to select some number of several available dishes
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
```

Comment:
Metrics/LineLength: Line is too long. [106/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)